src/main/java/org/reaktivity/nukleus/http_cache/internal/proxy/cache/DefaultCache.java

```diff
@@ -145,22 +171,74 @@ public boolean matchCacheableRequest(
         final DefaultCacheEntry cacheEntry = cachedEntries.get(requestHash);
         return satisfiedByCache(requestHeaders) &&
                 cacheEntry != null &&
-                cacheEntry.canServeRequest(requestHeaders, authScope);
-
+                cacheEntry.canServeRequest(requestHeaders, authScope) &&
+                !cacheEntry.isValid();
```

Comment:
Why `!isValid()`? Doesn't the cache entry need to be valid to match the cacheable request?

Reply:
it used to be `isValidationRequired` hence with `!` I have changed to `isValid()` method.

Reply:
Understood. I was asking because the other conditions in this `matchCacheableRequest` method seem to be positive about the cache entry, checking that it is not null, can serve the request, and at the end I was expecting it to check that the cache entry is valid, but instead it is ensuring the cache entry is not valid, why?

Reply:
sorry forgot to remove `!`. Thanks for catching it.